app/static/pages/volunteer/VolunteerInfoDefinitions.js

```diff
@@ -14,28 +14,13 @@
 
 oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
 
-  design: [{
-    fullName: 'Mark Halpin',
-    profilePictureImageFilename: 'mark.jpg',
-    professionalQuote: 'I contribute to Oppia because of its remarkable ' +
-      'goal: to make a quality education available to those who may not have ' +
-      'easy access to it.',
-    position: 'Design Lead'
-  }, {
-    fullName: 'Jordan Movish',
-    profilePictureImageFilename: 'jordan.jpg',
-    professionalQuote: 'As a former teacher, I am motivated to create a ' +
-      'platform that gives everyone equal access to education. Working with ' +
-      'Oppia is a powerful opportunity to leverage design for social justice.',
-    position: 'UX Designer'
-  }, {
-    fullName: 'Rachel Chen',
-    profilePictureImageFilename: 'rachel.jpg',
-    professionalQuote: 'Oppia provides more students with access to ' +
-      'individualized learning experiences. Volunteering for Oppia allows me ' +
-      'to contribute to this meaningful cause.',
-    position: 'UX Designer'
-  }],
+  artist: [{
```

Comment:
Should we change this to "Art" instead of "Artist? The other category indicates a certain field of expertise (development, local outreach, etc...)
The `position` property below should indicate clearly what the volunteer does within each specific category.

Reply:
done!